TimeInterval neglect_time
TimeInterval interaction_time

uint8 mode
uint8 NEGLECT_MODE = 0
uint8 INTERACTION_MODE = 1

float32 elapsed_time
